docs/examples/RESTCONF/index.md

```diff
@@ -0,0 +1,550 @@
+that would be returned for the comparable GET method, without the response message-body.
+It is supported for all resources that support the GET method.
+- The POST method is sent by the client to create a data resource.
+- The PUT method is sent by the client to create or replace the target data resource.
+- The DELETE method is used to delete the target resource.
+
+## EOS configuration
+
+The RESTCONF server is in the EOS device.
+
+### Generates a self-signed certificate
+
+```shell
+DC1-LEAF1A#security pki certificate generate self-signed restconf.crt key restconf.key generate rsa 2048 parameters common-name restconf
+certificate:restconf.crt generated
+DC1-LEAF1A#
+```
+
+### Change the default control-plane ACL
+
+The default RESTCONF port on Arista devices is TCP 6020.
+We need to change the default control-plane ACL on EOS in order to allow TCP 6020.
+
+```shell
+DC1-LEAF1A#show ip access-lists default-control-plane-acl
+```
+
+```shell
+DC1-LEAF1A(config)#show ip access-lists def2
+IP Access List def2
+        9 permit tcp any any eq 6020
+        10 permit icmp any any
+        20 permit ip any any tracked [match 147 packets, 0:00:14 ago]
+        30 permit udp any any eq bfd ttl eq 255
+        40 permit udp any any eq bfd-echo ttl eq 254
+        50 permit udp any any eq multihop-bfd
+        60 permit udp any any eq micro-bfd
+        70 permit udp any any eq sbfd
+        80 permit udp any eq sbfd any eq sbfd-initiator
+        90 permit ospf any any [match 1882 packets, 0:00:04 ago]
+        100 permit tcp any any eq ssh telnet www snmp bgp https msdp ldp netconf-ssh gnmi
+        110 permit udp any any eq bootps bootpc ntp snmp rip ldp
+        120 permit tcp any any eq mlag ttl eq 255
+        130 permit udp any any eq mlag ttl eq 255
+        140 permit vrrp any any
+        150 permit ahp any any
+        160 permit pim any any [match 124 packets, 0:00:14 ago]
+        170 permit igmp any any [match 90 packets, 0:01:24 ago]
+        180 permit tcp any any range 5900 5910
+        190 permit tcp any any range 50000 50100
+        200 permit udp any any range 51000 51100
+        210 permit tcp any any eq 3333
+        220 permit tcp any any eq nat ttl eq 255
+        230 permit tcp any eq bgp any
+        240 permit rsvp any any
+        250 permit tcp any any eq 6040
+```
+
+```shell
+DC1-LEAF1A(config)#sh run sec control
+system control-plane
+   ip access-group def2 vrf MGMT in
+```
+
+### Configure RESTCONF
+
+```shell
+DC1-LEAF1A(config)#sh run sec restconf
+management api restconf
+   transport https test
+      ssl profile restconf
+      vrf MGMT
+!
+management security
+   ssl profile restconf
+      certificate restconf.crt key restconf.key
+```
+
+### Verify
+
+```shell
+DC1-LEAF1A(config)#show management api restconf
+Enabled:            Yes
+Server:             running on port 6020, in MGMT VRF
+SSL Profile:        restconf
+QoS DSCP:           none
+```
```

Comment:
can reference the configuration in `docs/configuration/restconf.md`

Reply:
yep I will. 
we need to update docs/configuration/restconf.md to add CP ACL change

Reply:
done. also updated `docs/configuration/restconf.md` to add CP ACL change